.github/workflows/test.yml

```diff
@@ -30,3 +39,32 @@ jobs:
           python -m pip install .
       - name: Test
         run: python -m pytest -v
+
+  test-cpp:
+    name: Test-Cpp
+    runs-on: ubuntu-latest
+    needs: build-openassetio
+    container:
+      image: aswf/ci-vfxall:2022-clang14.3
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install Traitgen
+        run: python -m pip install git+https://github.com/feltech/OpenAssetIO-TraitGen@work/11-cppCodeGen
```

Comment:
[Fix] When a C++ traitgen release is available, update this.

Reply:
[Discuss] Have changed this to target main now, so this could technically go in and be fine, but I suspect we probably want to target a traitgen-release instead, which is todo.

https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/25/commits/24f2113784247f9ab91634450c948daabe9ef4d7

Reply:
https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/25/commits/3171dd099cdd70a0bbb2e1b4e2cdc311ad4283d6

and also 

https://github.com/OpenAssetIO/OpenAssetIO-MediaCreation/pull/25/commits/1e45e0d73c2ff923d3211f0b10b8c9bfeec4c23c